DSharpPlus.CommandsNext/CommandsNextExtension.cs

```diff
@@ -489,8 +489,19 @@ private void RegisterCommands(Type t, CommandGroupBuilder? currentParent, IEnume
                         foreach (var chk in inheritedChecks)
                             groupBuilder.WithExecutionCheck(chk);
 
-                        foreach (var mi in ti.DeclaredMethods.Where(x => x.IsCommandCandidate(out _) && x.GetCustomAttribute<GroupCommandAttribute>() != null))
-                            groupBuilder.WithOverload(new CommandOverloadBuilder(mi));
+                        if (ti.DeclaredMethods.Any(x => x.IsCommandCandidate(out _) && x.GetCustomAttribute<GroupCommandAttribute>() != null))
+                        {
+                            var groupCommandOverloads =
+                                ti.DeclaredMethods.Where(x => x.IsCommandCandidate(out _) && x.GetCustomAttribute<GroupCommandAttribute>() != null);
+
+                            // overloads don't work for group commands, so throw on startup instead of at runtime.
+                            if(groupCommandOverloads.Count() > 1)
+                            {
+                                throw new InvalidOverloadException("GroupCommands cannot be overloaded", groupCommandOverloads.First());
```

Comment:
`$"Commands marked with [{nameof(GroupCommandAttribute)}] cannot be overloaded"` may be more apt

Reply:
implemented